cmd/nerdctl/ipfs/ipfs_compose_linux_test.go

```diff
@@ -152,6 +153,7 @@ services:
       WORDPRESS_DB_USER: exampleuser
       WORDPRESS_DB_PASSWORD: examplepass
       WORDPRESS_DB_NAME: exampledb
+    # FIXME: this is flaky and will make the container fail on occasions
```

Comment:
What is "this"? The volume?

Reply:
Yes.

Reply:
On occasions php seem to hard fail trying to read a file from the volume that it is somehow expecting.

Reply:
Curious if the compose global lock PR will help with these.